modules/machineset-creating.adoc

```diff
@@ -48,17 +48,17 @@ metadata:
   creationTimestamp: 2019-02-15T16:32:56Z <1>
   generation: 1 <1>
   labels:
-    sigs.k8s.io/cluster-api-cluster: <cluster_name> <2>
-    sigs.k8s.io/cluster-api-machine-role: <machine_label> <3>
-    sigs.k8s.io/cluster-api-machine-type: <machine_label> <3>
+    machine.openshift.io/cluster-api-cluster:  <infrastructure_id> <2>
+    machine.openshift.io/cluster-api-machine-role: <machine_label> <3>
+    machine.openshift.io/cluster-api-machine-type: <machine_label> <3>
   name: <cluster_name>-<machine_label>-<AWS-availability-zone> <3> <4>
   namespace: openshift-machine-api
   resourceVersion: "9249" <1>
   selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/openshift-machine-api/machinesets/<cluster_name>-<machine_label>-<AWS-availability-zone> <1>
```

Comment:
should be `/apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machinesets/ <infrastructure_id>-<machine_label>-<AWS-availability-zone> `